SlevomatCodingStandard/Sniffs/Commenting/DeprecatedDocRequiresDescriptionSniff.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Commenting;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use const T_DOC_COMMENT_TAG;
+
+class DeprecatedDocRequiresDescriptionSniff implements Sniff
+{
+
+	public const MISSING_DEPRECATED_DESCRIPTION = 'MissingDeprecatedDescription';
+
+	/** @return array<int, (int|string)> */
+	public function register(): array
+	{
+		return [T_DOC_COMMENT_TAG];
+	}
+
+	/**
+	 * @phpcsSuppress SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint
+	 * @param File $phpcsFile
+	 * @param int $propertyPointer
+	 */
+	public function process(File $phpcsFile, $propertyPointer): void
+	{
+		$tokens = $phpcsFile->getTokens();
+
+		if ($tokens[$propertyPointer]['content'] !== '@deprecated') {
```

Comment:
Use `AnnotationHelper` please. It uses better parser.

Reply:
Sorry not familiar with this class.. but does it return the `deprecated`? https://github.com/slevomat/coding-standard/blob/master/SlevomatCodingStandard/Helpers/AnnotationHelper.php#L276

what's the object I should expect if I use the `getAnnotationsByName`?

Reply:
This class is part of this repository, so everything is in the code...